dali/util/random_crop_generator.cc

```diff
@@ -114,7 +114,7 @@ RandomCropGenerator::GenerateCropWindows(const TensorShape<>& shape,
   std::vector<CropWindow> crop_windows;
   for (std::size_t i = 0; i < N; i++) {
     rand_gen_.seed(seeds[i]);
-    crop_windows.push_back(GenerateCropWindowImpl(shape));
+    crop_windows.emplace_back(GenerateCropWindowImpl(shape));
```

Comment:
Please revert. The function returns exactly the element type and the push_back overload used is `push_back(T &&)`. The switch to `emplace_back` doesn't change anything and looks a bit foreign.